server/integration_tests/test_data/demo_feb2023/query_2/chart_config.json

```diff
@@ -11,7 +11,6 @@
                     "title": "Most severe fires in California",
                     "topEventTileSpec": {
                       "eventTypeKey": "fire",
```

Comment:
why "showEndDate" is deleted?

Reply:
we had really long events, and this was deleted in #2239. but the tests didn't catch that diff..